src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipeline.scala

```diff
@@ -4,5 +4,7 @@ import org.broadinstitute.monster.common.{PipelineBuilder, ScioApp}
 
 /** Entry-point for the Encode pipeline's Docker image. */
 object ExtractionPipeline extends ScioApp[Args] {
-  override def pipelineBuilder: PipelineBuilder[Args] = ExtractionPipelineBuilder
+
+  override def pipelineBuilder: PipelineBuilder[Args] =
```

Comment:
```suggestion
  override val pipelineBuilder: PipelineBuilder[Args] =
```
To ensure there's only ever 1 instance